app/views/citizens/accounts/index.html.erb

```diff
@@ -11,8 +11,48 @@
   We will show your statement from the last 30 days so you can tell us
   more about your financial situation
 </p>
+<div class="govuk-!-padding-bottom-5">
+  <h2 class="govuk-heading-m"><%= t '.your_accounts' %></h2>
+</div>
 
-<p>
-  <strong>Account holders's name</strong>
-  <%= @applicant.first_name %>
-</p>
+<% @applicant_banks.each do |applicant_bank| %>
+
+  <table class="govuk-table">
+    <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header" scope="row"><%= t '.acount_holder_name_heading' %></th>
+        <td class="govuk-table__cell"><%= applicant_bank.bank_account_holders.last.full_name %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header" scope="row"><%= t '.account_holder_address_heading' %></th>
+        <td class="govuk-table__cell"><%= applicant_bank.bank_account_holders.last.addresses.first.values.join(', ') %></td>
```

Comment:
**[not a blocker/suggestion]** Both here and line 24 we seems to be using very specific logic to display information on the page. 💭 Consider having a presenter (or helper methods, although I personally recommend the former) for the `applicant_bank` where `applicant_bank.bank_account_holders.last` could be the implementation for a **applicant_bank.main_bank_account_holder** for example, removing the chained logic from the view.